patterns/1-initial/extensions-to-manage-contributions-at-scale.md

```diff
@@ -0,0 +1,84 @@
+## Title
+
+Extensions to Manage Contributions at Scale
+
+## Patlet
+
+An InnerSource project is receiving too many contributions, making maintenance difficult and resulting in a large code review backlog or premature rejection of new feature contributions. 
+By offering an extension mechanism outside of the core project, the maintainers enable contributors to extend project capabilities with minimal code, thus encouraging innovation and experimentation. This would reduce the maintenance overhead on the core project, allowing it to contain only the strategic features that have been widely adopted, tested and validated. It would also ensure faster release of new features to the community, reducing the cost to harden a feature until it has proven its utility.
```

Comment:
@spier I edited the Patlet.

Reply:
nice!

It is a bit long now, as Palets are meant to be "problem/solution pairs in two sentences".
i.e. one sentence for the problem, and one for the solution.

The Patlet is what eventually will be shown in the [toc of the book](https://patterns.innersourcecommons.org/toc#p).
We try to keep them reasonably short to allow the readers to more quickly scan many patterns.

We could e.g. try to move some of the text from the Patlet into the Problem section.

However I would also be fine to leave this as is for now, and shorten the Patlet in future reviews.

You decide :)

Reply:
shortened it :)